src/Options.js

```diff
@@ -265,18 +290,18 @@ class Options {
      * Gets the CSS color of the current options object, accounting for the
      * `colorMap`.
      */
-    getColor() {
+    getColor(): ?string {
```

Comment:
If the `color` prop is defined as `color: string | void` on the class, then the return value would have to change to `string | void` here as well.